### ISSUE TITLE
Problem compiling PhantomJS using Docker

### ISSUE THREAD

**Issue by timmipetit** [ariya/phantomjs#14213]
1. Which version of PhantomJS are you using? master
2. What steps will reproduce the problem?
   1. Get a clean checkout of phantomjs
   2. Run `docker run -v $PWD:/src debian:wheezy /src/deploy/docker-build.sh`
   3. Wait
   4. It will eventually fail with this output:

```
Compiling PhantomJS...
Submodule '3rdparty-win' (https://github.com/Vitallium/phantomjs-3rdparty-win.git) registered for path 'src/qt/3rdparty'
Submodule 'qtbase' (https://github.com/Vitallium/qtbase.git) registered for path 'src/qt/qtbase'
Submodule 'qtwebkit' (https://github.com/Vitallium/qtwebkit.git) registered for path 'src/qt/qtwebkit'
Usage: git submodule [--quiet] add [-b branch] [-f|--force] [--reference <repository>] [--] <repository> [<path>]
   or: git submodule [--quiet] status [--cached] [--recursive] [--] [<path>...]
   or: git submodule [--quiet] init [--] [<path>...]
   or: git submodule [--quiet] update [--init] [-N|--no-fetch] [-f|--force] [--rebase] [--reference <repository>] [--merge] [--recursive] [--] [<path>...]
   or: git submodule [--quiet] summary [--cached|--files] [--summary-limit <n>] [commit] [--] [<path>...]
   or: git submodule [--quiet] foreach [--recursive] <command>
   or: git submodule [--quiet] sync [--] [<path>...]


ERROR: Failed to build PhantomJS! Initial update of git submodules failed.
Executing in /root/build/src: git submodule init
Executing in /root/build/src: git submodule update --remote
```
1. Which operating system are you using?
   OS X, using Docker 1.9.1

The problem seems to be this commit: cde33dc6fbae8c5d8950ee9b5336c8c66912dc64
If I build against 74ed9404c2ac70a100d6b9a666633b4e02568e94 everything works as expected

**Comment by timmipetit** [comment]
Compilation of `master` without commit cde33dc6fbae8c5d8950ee9b5336c8c66912dc64 also works as expected.

**Comment by ariya** [comment]
This was originally to fix #14082. Any comments @mepard?

**Comment by LiamKarlMitchell** [comment]
Thanks, reverting cde33dc6fbae8c5d8950ee9b5336c8c66912dc64 let me build 5232c9dff98115600d1b1491e538e01c0de2d650

Needed the DPI setting.

```
Prerequisites: git and docker.
Deploy instructions from here: https://github.com/ariya/phantomjs/tree/master/deploy

git clone https://github.com/ariya/phantomjs.git
cd phantomjs
git checkout 5232c9dff98115600d1b1491e538e01c0de2d650
git revert -n cde33dc6fbae8c5d8950ee9b5336c8c66912dc64
docker run -v $PWD:/src tubia/debian:wheezy /src/deploy/docker-build.sh
```
